signup/index.php

```diff
@@ -60,6 +60,15 @@
 							<label class="form-label form-required" for="phone">Phone Number</label>
 						</div>
 
+
+						<!-- TODO CHANGE THE ADD DEPENDENT TO SOMETHING MORE DESCRIPTIVE OF WHAT THIS IS FOR CLIENTS-->
```

Comment:
Is this TODO to be done in this PR or is there another Issue set up to take care of it?

Reply:
I think I would prefer to take care of it in this PR, however, the impact of doing it later is pretty minimal. All it involves in some simple renaming which is easy across the board (even in DB).